crates/aptos-telemetry-service/src/custom_telemetry.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use std::time::Duration;
+
+use crate::{
+    auth::with_auth,
+    context::Context,
+    types::auth::{Claims, TelemetryDump},
+};
+use aptos_config::config::PeerRole;
+use aptos_logger::{error, info};
+use gcp_bigquery_client::model::table_data_insert_all_request::TableDataInsertAllRequest;
+use serde_json::json;
+use warp::{filters::BoxedFilter, reject, reply, Filter, Rejection, Reply};
+
+pub fn custom_telemetry(context: Context) -> BoxedFilter<(impl Reply,)> {
+    warp::path!("telemetry")
```

Comment:
I think we should call this `custom_event` (incl. the path etc.) instead of telemetry.

Reply:
Changed to `custom_event`